test/sample.js

```diff
@@ -3,4 +3,16 @@ let log = require('logger'),
     lodash = require('lodash');
 
 let result = sortBy(_.flatten(uniq([{a: 1}, {a: 2}, {a: 1}, {a: 0}])), 'a');
-lodash.each(result, log);
\ No newline at end of file
+lodash.each(result, log);
+
+lodash.chain([1, 2, 3])
+ .tap(function(array) {
+   array.pop();
+ })
+ .map('x')
+ .reverse()
+ .value();
+
+_([1])
+ .isEmpty()
+ .toJSON();
```

Comment:
Mind sanity checking this output @jdalton 
`./bin/lodash-modularize test/sample.js --cjs`
```js
'use strict';

var lodash = require('lodash/chain/lodash.js');
var clone = require('lodash/object/clone.js');
var mixin = require('lodash/utility/mixin.js');


var each = require('lodash/collection/each');
var flatten = require('lodash/array/flatten');
var isEmpty = require('lodash/lang/isEmpty');
var map = require('lodash/collection/map');
var sortBy = require('lodash/collection/sortBy');
var uniq = require('lodash/array/uniq');


var __chain__ = require('lodash/chain/chain');
var __reverse__ = require('lodash/chain/reverse');
var __tap__ = require('lodash/chain/tap');
var __toJSON__ = require('lodash/chain/toJSON');
var __value__ = require('lodash/chain/value');

var _ = clone(lodash);

// Add the methods used through chaining and explict use
mixin(_, {
  each: each,
  flatten: flatten,
  map: map,
  sortBy: sortBy,
  uniq: uniq
}, true);

mixin(_, {
  isEmpty: isEmpty
}, true);


_.prototype.chain = __chain__;

_.prototype.reverse = __reverse__;

_.prototype.tap = __tap__;

_.prototype.toJSON = __toJSON__;

_.prototype.value = __value__;

module.exports = _;
```